content/v2.0/actions/mime-types-and-format.md

```diff
@@ -0,0 +1,170 @@
+---
+title: MIME Types and format
+order: 70
+---
+```ruby
+module MyApp
+  class App < Hanami::App
+    config.actions.format json: "application/json"
+    config.actions.default_request_format = :json
+    config.actions.default_response_format = :json
+    config.middleware.use :body_parser, :json
+  end
+end
+```
```

Comment:
This seems off, why there's no intro paragraph? Also, shouldn't we use the new shortcut now: `config.actions.format :json`?